forward_engineering/helpers/convertJsonSchemaToAvro.js

```diff
@@ -252,7 +242,7 @@ const convertFixed = schema => {
 	};
 
 	const schemaFromUdt = getUdtItem(name);
-	if (schemaFromUdt && _.isEqual(filterSchemaAttributes(convertedSchema), filterSchemaAttributes(schemaFromUdt))) {
+	if (schemaFromUdt && compareSchemasByStructure(filterSchemaAttributes(convertedSchema), filterSchemaAttributes(schemaFromUdt))) {
```

Comment:
changed comparator - the previous one was too strict. Anyway, if this check will not pass this will be a validation error because of two different types with the same names